server.go

```diff
@@ -490,7 +493,7 @@ func (s *Server) Get(def *Def, r io.Reader, w io.Writer) error {
 
 		// Exists. Write it out and go to the next one.
 		if err == nil {
-			fmt.Fprintln(w, alias)
+			fmt.Fprintln(w, "0", alias)
```

Comment:
I misunderstood, I thought "1" means "new record created" and "0" means the opposite